### ISSUE TITLE
Items disappearing above 1 stack

### ISSUE THREAD

**Issue by HimitsuYami** [jaquadro/StorageDrawers#549]
So I hooked up my mob farm drops to get automatically sorted into drawers, at first it seems all well and good until I notice items beginning to vanish. the drawer will hold one stack of items with no complaints (using a 2x2 drawer btw) if more go in they register but they start to slowly disappear and I can watch them being drained with WAILA and if I take them out there is indeed only the one stack left. I was wondering if someone might have an idea of what's causing it and how I can fix it. items are going in via an Ender IO Item Conduit going into an attached drawer controller. I have an Extra Utilities Transfer Node pulling out items with a durability bar (because they don't stack and would clog the system) but I've checked every attached inventory and the items aren't going anywhere they're just being voided. none of the drawers have any void upgrades though. after a little testing if the item count drops to 1 the drawer kills it as well

here's a screenshot while it's happening
![image](https://user-images.githubusercontent.com/21053554/28906266-52edaf52-77e4-11e7-90b4-5ab7f3c66a2b.png)

**Comment by jaquadro** [comment]
Does this only happen with the EU node attached?

**Comment by HimitsuYami** [comment]
I don't know anymore but I don't think so. I haven't played that world in a while and I just swapped out to either TE caches or JABBA barrels (I forget which)

**Comment by jaquadro** [comment]
Oh, this was a 1.7.10 problem :|

There's just a lot that could got wrong in 1.7.10-land.

**Comment by HimitsuYami** [comment]
Yeah but a lot of the mods I use haven't updated so I'm stuck :/ thanks for responding though

**Comment by tuxokc** [comment]
I've played Project Ozone over the past year or two, and have seen this happen frequently in all of that time.  I just tear down the controller and everything attached and rebuild it from scratch when it happens.

It can happen with compacting drawers as well, which is really frustrating when your loosing octupule compressed cobble stone!  

Even as I write this I'm looking at an Oak Drawers 2x2 with a Transfer Node with a filter for a random engineers hammer inserted.  It's the presence of that Transfer Node with a filter that seems to cause the cobble to vanish.

This is a revelation to me!  I hadn't given it so much thought until now.  The easy way to avoid this is to use a Retrieval Node at the other end of the path, which I've been doing all this time without realizing it was a viable work around to avoid item loss.

The question for the OP is did the Diamond chest get filtered items?

**Comment by Shad0wSlam** [comment]
Thanks for sharing that a Retrieval node works! Finally, I have Minicio. I'd forgotten they even existed!
